doc/source/dev/core-dev/github.rst.inc

```diff
@@ -41,26 +41,25 @@ Dealing with pull requests
 --------------------------
 
 - When merging contributions, a committer is responsible for ensuring that
-  those meet the requirements outlined in `Contributing to SciPy
-  <https://github.com/scipy/scipy/blob/main/doc/source/dev/hacking.rst>`_.
-  Also check
-  that new features and backwards compatibility breaks were discussed on the
-  scipy-dev mailing list.
+  those meet the requirements outlined in :ref:`Contributing to SciPy <hacking>`.
+  Also check that new features and backwards compatibility breaks were discussed
+  on the scipy-dev mailing list.
```

Comment:
I guess things have already changed since the PR was opened and this would be Discourse or whatever we call it.

Reply:
yep, let's keep the migration to Discourse for a separate PR